exercise/templatetags/exercise.py

```diff
@@ -128,6 +128,33 @@ def submission_status(status):
     return Submission.STATUS[status]
 
 
+"""
+    Used for adding relevant classes to the
+    points_badge to enable textual status.
+"""
+@register.simple_tag(takes_context=True)
+def get_points_class(context):
+    html_class = "" # holds result
+    if context['difficulty'] == "-" and context['max'] <= 2:
+        html_class = "aplus-points"
+        if (context['max'] == 2 and context['points'] == 2)\
+        or context['max'] == 1 and context['points']== 1:
+            html_class += " accepted"
+        elif (context['max'] == 2 and context['points'] == 1):
+            html_class += " ok"
+        elif context['points'] == 0 and context['submitted']:
+            html_class += " rejected"
```

Comment:
and probable classes `points` and `points-ok`. Just `ok` isn't very exact.

Reply:
Do you have any ideas as to what the middle step should be called in the 0-2 interval? 0-1 is pretty obvious (accepted/rejected).

Reply:
Rejected, ok, good. Actually, rejected is probably bad when displayed, so we probably need to find better word for it. For css/js/code it's relatively fine.